live-build/misc/upgrade-scripts/verify-impl

```diff
@@ -26,6 +26,25 @@ function usage() {
 	exit 2
 }
 
+function get_container_name() {
+	return basename $(df -k / | tail -1 | awk '{print $1}')
+}
+
+function cleanup() {
+	local rc="$?"
+
+	#
+	# Persist container logs by saving to the container's host.
+	# Delete and create verify log destination directory to save only one copy of log to limit
+	# space usage.
+	#
+	rm -rf "${UPDATE_VERIFY_LOG_DIR:?}"
+	mkdir -p "${UPDATE_VERIFY_LOG_DIR}/${get_container_name}"
+	${SUPPORT_INFO_SCRIPT} -o "${UPDATE_VERIFY_LOG_DIR}" -t "log dropbox"
```

Comment:
I think we might want to use `|| die "failure message"` for each of these commands.. if any of them fails, we want the script to return an error, right? as it is, I think the script could return success if any of these failed.